src/main/scala/mesosphere/marathon/core/storage/repository/impl/GroupRepositoryImpl.scala

```diff
@@ -0,0 +1,184 @@
+package mesosphere.marathon.core.storage.repository.impl
```

Comment:
Please move this class into the store module. The repository module should not have any knowledge about specific types.